Bucketeer/Sources/Public/BKTEvaluation.swift

```diff
@@ -19,3 +21,64 @@ public struct BKTEvaluation: Equatable {
         case prerequisite = "PREREQUISITE"
     }
 }
+
+public struct BKTEvaluationDetails<T:Equatable>: Equatable {
+    public let featureId: String
+    public let featureVersion: Int
+    public let userId: String
+    public let variationId: String
+    public let variationName: String
+    public let variationValue: T
+    public let reason: Reason
+
+    public enum Reason: String, Codable, Hashable {
```

Comment:
@duyhungtnn, WDYT of adding the `BKTEvaluationDetails` to a new file like you did in the Android SDK?

Reply:
It's good.
I updated https://github.com/bucketeer-io/ios-client-sdk/pull/90/commits/6e761aca5f0edf4453bcf9e5778f0d7d418916b8